fix: fixes based on feedback (#838)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sources/google-tag-manager-client.md

 
#### API Key

> Copy your amplitude project API Key in the API Key field. For EU residency, your project API Key is under analytics.eu.amplitude.com. Each project has different API Key, please make sure you are copy the API Key from the right project. Go to **Settings -> Projects -> click the right project name from the list** to find your project API Key. 

Reviewer: Maybe make `analytics.eu.amplitude.com` a link? Or call out using "code style" with tikmarks?

Reviewer 2: How about adding the link https://www.docs.developers.amplitude.com/analytics/find-api-credentials/?

#### identify

!!!note 

>     Identify calls will not be showing like a single event in user look up. The identify calls will set the user properties, and those updated user properties will appear only after the next event fired by the User.

Reviewer: ```suggestion
    Identify calls _don't_ appear in user look up. The identify calls will set the user properties, and those updated user properties will appear only after the next event fired by the User.
```
